test/test_torch.py

```diff
@@ -938,6 +939,240 @@ def test_add(self):
 
         # [res] torch.add([res,] tensor1, value, tensor2)
 
+    def mb_rand(self, *dims):
+        dims = [dim for dim in dims if dim != ()]
+        xs = [torch.rand(1, *(random.randint(1, size) if b else size for b, size in dims[1:])) for i in range(dims[0])]
+        xb = MaskedBatch.fromlist(xs, tuple(b for b, d in dims[1:]))
+        return xs, (xb.data, xb.mask, xb.dims)
+
+    def mb_assert_allclose(self, xs, ybs):
+        if isinstance(ybs, torch.autograd.Variable):
+            self.assertAlmostEqual(xs.data, ybs.data, places=3)
+        elif isinstance(ybs, MaskedBatch):
+            self.mb_assert_allclose(xs, ybs.examples())
+        else:
+            if isinstance(ybs, (list, tuple)):
+                for j, yb in enumerate(ybs):
+                    for i, y in enumerate(yb.examples()):
+                        self.mb_assert_allclose(xs[i][j], y)
+            else:
+                for x, yb in zip(xs, ybs):
+                    self.mb_assert_allclose(x, yb)
+
+    def test_maskedbatch_elementwise_unary_ops(self):
+        ops_list = ['tanh', 'relu', 'sigmoid']
+        for op in ops_list:
+            xs, batch = self.mb_rand(4, (True, 3), (False, 2))
+            ys = [eval('torch.' + op)(xs[j]) for j in range(4)]
+            ybs = eval('torch.b_' + op)(batch)
```

Comment:
It would be much nicer and safer to implement this as `getattr(torch, 'b_' + op)(batch)` instead of using `eval`.